docs/source/learn/eager.md

```diff
@@ -24,24 +24,40 @@ res = model(input)
 torch_xla.sync()
 ```
 
-The actual model compilation and device execution happens when
-`torch_xla.sync` is called. There are multiple drawback of this
-approach.
+While this approach enables performance optimizations, it introduces significant
+usability challenges.
 
-1.  Users are often confused about when the framework is tracing and
-    when the framework is executing.
-2.  Non-core model code(data preprocessing for example) often generates
-    some small pending execution that gets leaked into the main
-    graph(step function) and causes recompilation. The recompilation of
-    the whole graph is usually very expensive.
-3.  It is hard to debug when/why recompilation happens.
+## Challenges with LazyTensor Mode
 
-To mitigate above issues we want to introduce the new UX with eager and
-compile.
+- **Ambiguity**: Developers struggle to distinguish between tracing and
+execution phases, complicating development and debugging.
 
-## Basic Usage
+- **Recompilation Overhead**: Non-core operations (e.g., data preprocessing) can
+leak into the graph, triggering expensive recompilations.
 
-``` python
+- **Debugging Difficulty**: Identifying the cause of recompilations is
+challenging due to the opaque nature of graph-building processes.
+
+## Eager Mode and `torch_xla.compile`
+
+To address these issues, PyTorch/XLA introduces an experimental eager mode
+(enabled via `torch_xla.experimental.eager_mode(True)`) and the
+`torch_xla.compile` API. **This shift aligns PyTorch/XLA more closely with
```

Comment:
Did you mean to emphasize this sentence? Seems a bit jarring to me but just a minor opinion.

Reply:
I see you point. This can go un-bolded. Instead used emphasize to call out the fact that `torch_xla.compile` is useful independent of eager mode as well.